feedstocks.html.tmpl

```diff
@@ -65,7 +65,7 @@
             <div class="col-lg-8 col-lg-offset-2">
                 <h2 class="text-center">Feedstocks on conda-forge</h2>
                 <ul class="list-group">
-                  {% for feedstock in gh_feedstocks %}
+{% for feedstock in gh_feedstocks %}
```

Comment:
After jinja processes this, it leaves the whitespace from this around. Figure it is simpler to de-dent to avoid this as it will parse it the same.